README.en.md

```diff
@@ -0,0 +1,132 @@
+# Cloud-Barista's Coffeehouse
+
+Explanation of different perspectives lowers the entry barriers for future contributors. 😉
+
+Read this in other languages: [English](https://github.com/cloud-barista/cb-coffeehouse/blob/main/README.en.md) or [한국어](https://github.com/cloud-barista/cb-coffeehouse/blob/main/README.md).
+
+> The historian Brian Cowan describes English coffeehouses as "places where people gathered to drink coffee, learn the news of the day, and perhaps to meet with other local residents and discuss matters of mutual concern.“ (See [English coffeehouses in the 17th and 18th centuries](https://en.wikipedia.org/wiki/English_coffeehouses_in_the_17th_and_18th_centuries))
+
+English Coffeehouse (aka Penny House) was a social club where intellectuals shared various opinions while sipping a cup of coffee for 1 penny.
+
+I hope that Cloud-Barista’s Coffeehouse can share various information “easily” like this.
+
+**<ins>Participate in open source projects by freely sharing information, explanations, and opinions, naturally find contribution points in Cloud-Barista's various Repositories, and become a Contributor, Reviewer, and Maintainer!**</ins> 😆
```

Comment:
```suggestion
**<ins>Cloud-Barista community encourages you to freely share information, comments, and opinions. This may be the first step towards getting involved in an open source project. In this process, you could find contribution points in Cloud-Barista's various repositories, and become a Contributor, Reviewer, and Maintainer naturally !**</ins> 😆
```